README.zh.md

```diff
@@ -0,0 +1,124 @@
+# Crouton中文版教程
+
+> 基于crouton项目README的英文版进行汉化，部分内容没有进行汉化，建议有能力者优先阅读英文版本。
+
+## 简介
+
+​	Chroot是Chromium OS Universal Chroot Environment 的简写，是一系列脚本的合集，利用Linux的Chroot，在Chromebook上同时运行Chrome OS和某个Linux发行版。
+
+## Chroot介绍
+
+​	Chroot命令用来在指定的根目录下运行指令。Chroot的这种功能可以为第二系统提供一个隔离的文件系统，就像虚拟化一样，但是第二系统实际上仍然在主系统的文件系统下面工作，在进程和网络层面，chroot并没有进行隔离。
+
+​	至于详细的内容，为什么不去问问[百度搜索](https://www.baidu.com/s?wd=chroot)？

```

Comment:
Maybe a Wikipedia link instead? https://zh.wikipedia.org/wiki/Chroot

Reply:
```suggestion
​	至于详细的内容，为什么不去问问[维基百科](https://zh.wikipedia.org/wiki/Chroot)？（或者[百度百科](https://www.baidu.com/s?wd=chroot)，如果你上维基很困难的话）

```
A Wikipedia link is a good idea. Wanna keep the Baidu link cause one may have difficulty accessing Wikipedia in mainland China.

Reply:
noted.